src/web/app/components/question-edit-form/question-edit-form.component.html

```diff
@@ -277,6 +264,21 @@ <h5>Changing the visibility after collecting responses is not recommended.</h5>
     </div>
   </div>
 </div>
+<ng-template #discardModal let-modal>f
```

Comment:
- `f`?

- Since we are here. I think we can just create new component for the modal instead of appending it here.

Reply:
>     * Since we are here. I think we can just create new component for the modal instead of appending it here.

I think can deter to UI / UX improvements? I think there are plans to standardize the modals.

Reply:
Ok